config/config.go

```diff
@@ -6,27 +6,48 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"regexp"
+	"strings"
 
 	"github.com/99designs/gqlgen/codegen/config"
 	"github.com/Yamashou/gqlgenc/client"
 	"github.com/Yamashou/gqlgenc/introspection"
+	"github.com/pkg/errors"
+	"github.com/vektah/gqlparser/v2"
 	"github.com/vektah/gqlparser/v2/ast"
 	"github.com/vektah/gqlparser/v2/validator"
 	"golang.org/x/xerrors"
 	"gopkg.in/yaml.v2"
 )
 
+// Config extends the gqlgen basic config
+// and represents the config file
 type Config struct {
-	Model    config.PackageConfig `yaml:"model,omitempty"`
-	Client   config.PackageConfig `yaml:"client,omitempty"`
-	Models   config.TypeMap       `yaml:"models,omitempty"`
-	Endpoint EndPointConfig       `yaml:"endpoint"`
-	Query    []string             `yaml:"query"`
+	SchemaFilename StringList           `yaml:"schema,omitempty"`
+	Model          config.PackageConfig `yaml:"model,omitempty"`
+	Client         config.PackageConfig `yaml:"client,omitempty"`
+	Models         config.TypeMap       `yaml:"models,omitempty"`
+	Endpoint       *EndPointConfig      `yaml:"endpoint,omitempty"`
```

Comment:
Why did you use it as a pointer?

Reply:
Did you want to prove your existence?

https://github.com/Yamashou/gqlgenc/pull/20/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R100-R104

Reply:
> Did you want to prove your existence?
> 
> https://github.com/Yamashou/gqlgenc/pull/20/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R100-R104

Hi @Yamashou, not sure what you are asking, hope not to misunderstand.
In my opinion we have to check if local schemas or remote endpoint is specified, using a pointer and `omitempty` directive I can reach that goal simply checking if `Endpoint` is nil

Reply:
I understand！
Thanks!